src/System.Security.Cryptography.OpenSsl/src/Resources/Strings.resx

```diff
@@ -132,6 +132,12 @@
   <data name="Cryptography_HashAlgorithmNameNullOrEmpty" xml:space="preserve">
     <value>The hash algorithm name cannot be null or empty.</value>
   </data>
+  <data name="Cryptography_CurveNotSupported" xml:space="preserve">
+    <value>The specified curve '{0}' or its parameters are not valid for this platform.</value>
+  </data>
+  <data name="Cryptography_InvalidKeySize" xml:space="preserve">
```

Comment:
Cryptography_InvalidKeySize is already in use elsewhere.  I have no idea if our loc process is going to get angry for a resource key with a different value; but it's definitely confusing to me.

Cryptography_InvalidEcKeySize seems better to me.